### PR TITLE
Add mindurka to V7 list

### PR DESCRIPTION
**Pull request by Darkness6030**

### CODE REVIEW

File: servers_v7.json [hunk 1]
```diff
@@ -9,5 +9,9 @@
   {
     "name": "Omega",
     "address": "157.90.180.53:25777"
+  },
+  {
+    "name": "Mindurka",
+    "address": "mindurka.tk, mindurka.tk:4000"
```
Comment (Quezler) [inline servers_v7.json:15]:
```suggestion
    "address": ["mindurka.tk, mindurka.tk:4000"]
```

**Comment by Ademun** [comment]
:)

**Comment by osp54** [comment]
@Anuken why ignore

**Comment by Anuken** [comment]
Read the suggestion above. Your JSON is invalid.

**Comment by WilloIzCitron** [comment]
@Darkness6030 fix your JSON, your json is invalid

**Comment by azmuth-slime** [comment]
Stop

**Comment by Darkness6030** [comment]
@Anuken  im so sorry, i fixed

File: servers_v7.json [hunk 2]
```diff
@@ -9,5 +9,9 @@
   {
     "name": "Omega",
     "address": "157.90.180.53:25777"
+  },
+  {
+    "name": "Mindurka",
+    "address": ["mindurka.tk, mindurka.tk:4000"]
```
Comment (Anuken) [inline servers_v7.json:15]: This still isn't valid, the IPs aren't separated by quotes.

**Comment by Darkness6030** [comment]
@Anuken sry again :-/

**Pull request merged**